### PR TITLE
Integrated performance/Utilzation card

### PR DESCRIPTION
**Pull request by gnehapk**
ref: https://github.com/kubevirt/web-ui-components/pull/330

### CODE REVIEW

**Comment by gnehapk** [comment]
@rawagner @mareklibra Please review.

File: frontend/public/storage/components/storage-overview/storage-overview.jsx [hunk 1]
```diff
@@ -24,6 +24,11 @@ const STORAGE_CEPH_CAPACITY_USED_QUERY = 'ceph_cluster_total_used_bytes';
 const CEPH_OSD_UP_QUERY = 'sum(ceph_osd_up)';
 const CEPH_OSD_DOWN_QUERY = 'count(ceph_osd_up == 0.0) OR vector(0)';
 
+const UTILIZATION_IOPS = '(sum(rate(ceph_pool_wr[1m])) + sum(rate(ceph_pool_rd[1m])))[360m:5m]';
```
Comment (rawagner) [inline frontend/public/storage/components/storage-overview/storage-overview.jsx]: please add `QUERY` to all constants

Comment (gnehapk) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: ack

Comment (gnehapk) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: @rawagner incorporated the comments.

**Pull request merged**